docs/user/openstack/vgpu.md

```diff
@@ -0,0 +1,456 @@
+# Installing a cluster on OpenStack with vGPU support
+
+If the underlying OpenStack deployment have proper GPU hardware installed and
+configured there is a way for passing down vGPU to the pods by using
+gpu-operator.
+
+
+## Pre-requisites
+
+The following steps are required to be checked before starting the deployment of
+OpenShift.
+
+- Appropriate hardware is installed (like [Nvidia Tesla
+  V100](https://www.nvidia.com/en-gb/data-center/tesla-v100)) on
+  the OpenStack compute node
+- NVIDIA host drivers installed and removed nouveau driver
+- Compute service installed on it and properly configured
+
+
+## Driver installation
+
+All of the examples assumes RHEL8 is used.
+
+Given, there is NVIDIA vGPU capable card installed on the machine which intended
+to have compute role, which may be confirmed by using a command which should
+display similar output:
+
+```console
+$ lspci -nn | grep -i nvidia
+3b:00.0 3D controller [0302]: NVIDIA Corporation GV100GL [Tesla V100 PCIe 16GB]
+[10de:1db4] (rev a1)
+```
+make sure to remove `nouveau` driver from loading. There might be necessary to
+add it to `/etc/modprobe.d/blacklist.conf` and/or changing
+grub config:
+
+```console
+$ sudo sed -i 's/console=/rd.driver.blacklist=nouveau console=/' /etc/default/grub
+$ sudo grub2-mkconfig -o /boot/grub2/grub.cfg
+```
+
+After that install host vGPU NVIDIA drivers:
+
+```console
+$ sudo rpm -iv NVIDIA-vGPU-rhel-8.4-510.73.06.x86_64.rpm
```

Comment:
Should we point where to get the RPM from?

Reply:
From NVIDIA app hub?

Reply:
Oh yes.